### ISSUE TITLE
focusAutofocus directive really need a child scope?

### ISSUE THREAD

**Issue by tolemac** [obogo/angular-focusmanager#17]
I think `focusAutofocus` directive don't need a child scope to work, creating a new scope you hide ng-model property for element scope.

You can see an example: http://codepen.io/jros/pen/doryRL?editors=101

**Comment by roboncode** [comment]
Thanks for the example. I am reviewing it and will make the changes needed for it to work.

**Comment by roboncode** [comment]
I have updated the example you provided. The example did not represent a real-world implementation of an Angular application. Models (not properties) are used when binding data to a form field or another data manipulating element. Therefore there will always be a .(dot) in the bind. This was stated and reemphasized several times by the Angular team and other web professionals many years back.

http://stackoverflow.com/questions/18128323/angularjs-if-you-are-not-using-a-dot-in-your-models-you-are-doing-it-wrong

You would find that this example would break with many different directives out there, not just "focusAutofocus".

I have updated the example to demonstrate the means to making this work...

http://codepen.io/anon/pen/gpNGRW?editors=101

Rob
